### PR TITLE
Initialise React app w/ redux template

### PR DESCRIPTION
**Pull request by disc3110**
- Initialise React app.
- Structure your application files using a "feature folder"
- Add React Router and set two <Route>s and <Link>s for the app's navigation: Books and Categories

### CODE REVIEW

File: src/components/CreateBook.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
```
Comment (arslanbisharat) [inline src/components/CreateBook.js:2]: - [ ] For now please only have book.js in component directory. There should no other file.

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import '../style/Header.css';
+// import { Link } from 'react-router-dom';
```
Comment (arslanbisharat) [inline src/components/Header.js]: - [ ] Please remove commented code to write clean code.

File: src/components/ListOfBooks.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
Comment (arslanbisharat) [inline src/components/ListOfBooks.js:1]: - [ ] For now please only have book.js in component directory. There should no other file.

File: src/redux/books/books.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/* import { createStore } from 'redux';
```
Comment (arslanbisharat) [inline src/redux/books/books.js]: - [ ] Why this file has commented code only?

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/redux/books/books.js (hunk 4 above)
Comment (disc3110) [reply src/redux/books/books.js]: Sorry, I started working on the redux but then I realized it was a duty for the next milestone

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**